app/views/layouts/application.haml

```diff
@@ -31,6 +31,8 @@
                     = link_to 'FAQ', faq_path
                   %li{ class: active_link('/schedule') }
                     = link_to 'Schedule', schedule_path
+                    %li{ class: active_link('/attendees') }
```

Comment:
It looks like this is indented 1 too many and I believe that would make it nested within schedule.